packages/runtime/src/next/components/tests/controls/page-control-prop-rendering.tsx

```diff
@@ -86,33 +130,58 @@ export async function testPageControlPropRendering<D extends ControlDefinition>(
       )
     }),
     {
-      type: TestComponentType,
-      label: 'TestComponent',
+      ...testComponentMeta,
       props: {
         propKey: controlDefinition as any,
       },
     },
   )
 
-  // Assert
-  await act(async () =>
-    render(
-      <Testing.ReactProvider runtime={runtime}>
-        <Page snapshot={snapshot} />
-      </Testing.ReactProvider>,
-    ),
+  const testElementTree = (component: ReactNode) => (
+    <Testing.ReactProvider runtime={runtime}>{component}</Testing.ReactProvider>
   )
 
-  if (action) {
-    await act(async () => {
-      await action(screen.getByTestId(testId))
-    })
-  }
-
-  expect(snapshot).toMatchSnapshot('snapshot')
-  expect(propSnapshot(screen.getByTestId(testId))).toMatchSnapshot('resolvedValue')
-
-  if (expectedRenders != null) {
-    expect(Number(screen.getByTestId(renderCountTestId).textContent)).toBe(expectedRenders)
+  if (!isServer()) {
```

Comment:
`isServer()` will return false if we are running in Jest's `jsdom` browser environment.